libdrgn/linux_kernel.h

```diff
@@ -30,12 +31,21 @@ linux_kernel_report_debug_info(struct drgn_program *prog,
 
 #ifdef WITH_LIBKDUMPFILE
 struct drgn_error *drgn_program_set_kdump(struct drgn_program *prog);
+struct drgn_error *drgn_program_prstatus_kdump_iter(struct drgn_program *prog,
+						     drgn_prstatus_callback_fn cb_fn);
 #else
 static inline struct drgn_error *
 drgn_program_set_kdump(struct drgn_program *prog)
 {
         return drgn_error_create(DRGN_ERROR_INVALID_ARGUMENT,
-	                         "drgn was built without libkdumpfile support");
+				  "drgn was built without libkdumpfile support");
+}
+static inline struct drgn_error *
+drgn_program_prstatus_kdump_iter(struct drgn_program *prog,
```

Comment:
The only call to this is gated behind `#ifdef WITH_LIBKDUMPFILE`, so I don't think we need the dummy definition, right?